frontend/src/lib/constants.ts

```diff
@@ -16,11 +16,13 @@ export const PIE_CHART_LABEL = 'Pie'
 export enum OrganizationMembershipLevel {
     Member = 1,
     Admin = 8,
+    Owner = 15,
 }
 
 export const organizationMembershipLevelToName = new Map<number, string>([
```

Comment:
lib/constants are super generic.

I wonder if this + OrganizationMembershipLevel makes more sense under `src/scenes/organization/levels.ts` since it keeps all orgs-related code under `scenes/organizations`

Reply:
I'll keep it where it now.